src/main.js

```diff
@@ -101,7 +101,10 @@ function createWindow() {
         // It's best if the dashboard takes up as much space as possible without covering the DriverStation application.
         // The window is closed until the python server is ready
         show: false,
-        icon: __dirname + '/../images/icon.png'
+        icon: __dirname + '/../images/icon.png',
+        webPreferences: {
+            nodeIntegration: true
+        }
```

Comment:
This is necessary because the connections file uses code from the ui file. There's currently an error in console when the robot connects because of this.

Reply:
Ah, so just new Electron features. Cool!